VideoLocker/res/layout/navigation_drawer_container.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/slider_menu"
+    android:layout_width="304dp"
+    android:layout_height="match_parent"
+    android:layout_gravity="left|start"
```

Comment:
Actually, that gravity is needed. If it's not set the drawer does not work properly. The navigation drawer toggle looks for the explicitly set gravity of the drawer.